modules/mocked_authentication/lib/mock_credential/service.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+require 'sign_in/logingov/service'
+require 'sign_in/idme/service'
+
+module MockedAuthentication
+  module MockCredential
```

Comment:
This isn't a big deal but what if we remove `Mock` from all these nested namespaces? It seems kinda redundant to have `MockedAuthentication::Mock*` when it could be inferred that `MockedAuthentication::Credential` is the mocked credential. Thoughts?

Reply:
Ya, might as well. I removed Mock from the classes in this engine